encoding/wkt/wktDecode.peg

```diff
@@ -0,0 +1,58 @@
+{
+package wkt
+
+func toIfaceSlice(v interface{}) []interface{} {
+    if v == nil {
+        return nil
+    }
+    return v.([]interface{})
+}
+}
+
+Input <- expr:Expr EOF {
+    cntCodeBlocks++
+    return expr, nil
+}
+
+Expr <- first:GeomClass _ empty:( "EMPTY" )* _ rest:( Point / MultiPoint) {
+    return rest, nil
+}
+
+GeomClass <- exp:( "POINT" / "MULTIPOINT" )  {
+    return exp, nil
+}
+
+Point <- _ '(' _ first:( Number ) _ second:( Number ) _ ')' {
+    return geom.Point{first.(float64), second.(float64)}, nil
+}
+
+MultiPoint <- _ '(' _ vals:(( Point ) _ [,]? )* ')' {
+    geo := make([][2]float64, 0)
```

Comment:
I know this needs to go in the top section and made generic but is this what you were thinking?